oshdb-api/src/main/java/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.java

```diff
@@ -488,10 +488,9 @@ protected MapReducer(MapReducer<?> obj) {
       ret.filters.add(ignored -> false);
       return ret;
     }
-    int keyId = keyValueId.getKey();
-    int valueId = keyValueId.getValue();
+    OSHDBTagKey keyId = new OSHDBTagKey(keyValueId.getKey());
     ret.preFilters.add(oshEntitiy -> oshEntitiy.hasTagKey(keyId));
-    ret.filters.add(osmEntity -> osmEntity.hasTagValue(keyId, valueId));
+    ret.filters.add(osmEntity -> osmEntity.hasTagValue(keyValueId.getKey(), keyValueId.getValue()));
```

Comment:
Sorry, I accidentally committed this, since it isn't actually directly related to this PR. But it makes the methods (`osmTag(key)` and `osmTag(key, value)`) work more similar in the way they use the `preFilter` function.